checkbox-provider-kivu/bin/gpu-load.py

```diff
@@ -42,8 +43,48 @@ def load_intel(fname : str, timeout : int) -> float:
 
     return compute_avg_intel(fname)
 
+def get_amd_vcn():
+    """
+    Open /sys/kernel/debug/dri/0/amdgpu_pm_info
+    and check for VCN (Video Core Next) activity
+    Return:
+      - -1 : if VCN is not present or error
+      -  0 : if VCN is disabled
+      - 100: if VCN is used
+    """
+    load = -1
+    try:
+        with open('/sys/kernel/debug/dri/0/amdgpu_pm_info', 'r') as f:
+            file_content = f.read()
+            if 'VCN: Enabled' in file_content:
+                load = 100
+            elif 'VCN: Disabled' in file_content:
+                load = 0
+            else:
+                load = -1
+    except OSError:
+        print('Error reading /sys/kernel/debug/dri/0/amdgpu_pm_info')
+        pass
+    return load
+
+def load_amd_vcn(timeout: int):
+    total = 0
+    remaining_time = timeout
```

Comment:
there's no need for another copy of that value, timeout can be modified directly, it's copied to the function

Reply:
timeout is used later on to compute average value ....

Reply:
Ah, indeed, I missed it, sorry!